### PR TITLE
CONN:3319-Updated Carbon Black Cloud App doc

### PR DESCRIPTION
**Pull request by parth-sumo**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/security-threat-detection/carbon-black-cloud.md [hunk 1]
```diff
@@ -2,19 +2,18 @@
 id: carbon-black-cloud
 title: Carbon Black Cloud
 sidebar_label: Carbon Black Cloud
-description: The Carbon Black Cloud app analyzes alert and event data from the Endpoint Standard and Enterprise EDR products. app dashboards provide visibility into threats, TTPs, devices, and more.
+description: The Carbon Black Cloud app analyzes alert and event data from the Endpoint Standard and Enterprise EDR products. App dashboards provide visibility into threats, TTPs, devices, and more.
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/carbon-black-cloud.md:5]:
```suggestion
description: The Carbon Black Cloud app analyzes alert and event data from the Endpoint Standard and Enterprise EDR products. 
```

File: static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Alerts.png
Comment (JV0812) [inline static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Alerts.png:1]: Can you please crop the image and remove the extra space in the image.

File: static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Overview.png
Comment (JV0812) [inline static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Overview.png:1]: Can you please crop the image and remove the extra space in the image.

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor changes.

File: static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Overview.png
Comment (parth-sumo) [reply static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Overview.png:1]: Done

File: static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Alerts.png
Comment (parth-sumo) [reply static/img/integrations/security-threat-detection/Carbon-Black-Cloud-Endpoint-Standard-Alerts.png:1]: Done

File: docs/integrations/security-threat-detection/carbon-black-cloud.md (hunk 1 above)
Comment (parth-sumo) [reply docs/integrations/security-threat-detection/carbon-black-cloud.md:5]: Done

**Review by JV0812 — COMMENTED** [review]
LGTM

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**